src/PrestaShopBundle/Resources/views/Admin/macros.html.twig

```diff
@@ -36,10 +36,14 @@
   </span>
 {% endmacro %}
 
-{% macro infotip(text)%}
+{% macro infotip(text, raw = false)%}
```

Comment:
```suggestion
{% macro infotip(text, use_raw = false)%}
```

To be sure there is zero conflict between `|raw` and the variable.

Reply:
@PierreRambaud done !